pkg/extensions/search/cve/cve.go

```diff
@@ -18,15 +18,15 @@ import (
 type CveInfo interface {
```

Comment:
IMHO we should keep the Scanner/CveInfo logic per platform and not handle image indexes if possible.
It complicates the logic and we're not sure how it's going to eventually work, since trivy doesn't support what we need yet.